### PR TITLE
[ADD]Add the done button and update its method in hotel folio.

### PR DESCRIPTION
**Pull request by KrishnaPrajapati-SerpentCS**

### CODE REVIEW

File: hotel/models/hotel.py [hunk 1]
```diff
@@ -601,12 +601,8 @@ def action_done(self):
...
         @param self: object pointer
         '''
-        sale_order_obj = self.env['sale.order']
-        res = False
-        for o in self:
-            sale_obj = sale_order_obj.browse([o.order_id.id])
-            res = sale_obj.action_done()
-        return res
+        for sale in self:
+            sale.write({'state': 'done'})
```
Comment (JayVora-SerpentCS) [inline hotel/models/hotel.py]: I think its good to call action_done if the method exists.

Comment (JayVora-SerpentCS) [reply hotel/models/hotel.py]: self.action_done() - one line will do everything, no loop needed.

File: hotel/views/hotel_view.xml [hunk 2]
```diff
@@ -492,6 +492,7 @@
                      <button name="action_cancel_draft" states="cancel" string="Set to Draft" type="object" icon="gtk-file" class="oe_highlight"/>
                      <button name="action_cancel" string="Cancel Folio" states="draft,manual,progress" type="object" icon="gtk-cancel" class="oe_highlight"/>
                      <button name="invoice_cancel" string="Cancel Folio" states="invoice_except" icon="gtk-cancel" class="oe_highlight"/>
+                     <button name="action_done" string="Set to Done" states="progress" type="object" icon="gtk-apply" class="oe_highlight"/>
```
Comment (JayVora-SerpentCS) [inline hotel/views/hotel_view.xml]: manual +progress na hovu joie?

Comment (JayVora-SerpentCS) [reply hotel/views/hotel_view.xml]: And Set to draft ma cancel +  done?

Comment (KrishnaPrajapati-SerpentCS) [reply hotel/views/hotel_view.xml]: When folio is done state then all process are done so not need to set as a draft state.

File: hotel/models/hotel.py [hunk 3]
```diff
@@ -603,9 +603,8 @@ def action_done(self):
         '''
         sale_order_obj = self.env['sale.order']
         res = False
-        for o in self:
-            sale_obj = sale_order_obj.browse([o.order_id.id])
-            res = sale_obj.action_done()
+        sale_obj = sale_order_obj.browse([self.order_id.id])
+        res = sale_obj.action_done()
         return res
```
Comment (JayVora-SerpentCS) [inline hotel/models/hotel.py]: Cannot this work?:
return self.order_id.action_done()

**Comment by JayVora-SerpentCS** [comment]
Please fix travis

**Pull request merged**